### DISCUSSION TITLE
Legendary as a ~~runner~~

### DISCUSSION THREAD

**Discussion by Cuteistfox** [lutris/lutris#5480] [Ideas]
[Legendary](https://github.com/derrod/legendary) is a reimplementation of epic games launcher that is way more lightweight(ram and cpu and storage space) and is free software! Anyways,i will keep runing it by manuly setting it to execute via the "linux" runner

**Comment by strycore** [comment]
It's been 5 years since Epic Games has given the EpicMegagrant to the Lutris project so maybe we can reconsider our position, especially since we're going to implement similar solutions for GOG and EA

**Reply by Cuteistfox** [reply]
At least in the arch build of lutris, a gog auto-downloader already exists.

**Reply by strycore** [reply]
Our GOG implementation is slow and doesn't allow for incremental updates.

It's going to get much better soon.

**Comment by DocMAX** [comment]
+1

**Comment by sigzegv** [comment]
SO, is there any news here ? The legendary runner is a really good idea to implement as it allows to remove the need of a launcher. (And then all the epic games in the same prefix ..)

**Comment by strycore** [comment]
Well I'm writing the EGS plugin for Playtron and hopefully it gets open sourced so we'll be able to use the Steam, Epic and GOG plugins in Lutris.
But I doubt it will change the fact that using the launcher is much more reliable. Using a single prefix for multiple games isn't a problem, that's just an old meme from 2010 back when Wine wasn't what it was today. 
What's funny is that people who say that today are incapable of giving a single example of 2 games conflicting with each other when grouped in a single prefix.

**Reply by sigzegv** [reply]
well I agree, but actually I have a game on steam called "car city driving" that doesn't work with anything else than Proton 8, so of course it's a rare case but it's appreciable to be able to split prefixes sometimes.
But anyways, asking about legendary support does not necessarily correlate with the prefixes argument.

**Reply by strycore** [reply]
Just because a bunch of games share a same prefix doesn't mean you can't change the Proton version
